charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl

```diff
@@ -69,3 +69,49 @@
 {{- end }}
 {{- end }}
 {{- end -}}
+
+{{- define "kube-apiserver.auditversion" -}}
+{{- if semverCompare ">= 1.12" .Values.kubernetesVersion -}}
+audit.k8s.io/v1
+{{- else -}}
+audit.k8s.io/v1beta1
+{{- end -}}
+{{- end -}}
```

Comment:
If you want to keep that section in this file then please remove it from [here](https://github.com/gardener/gardener/blob/master/charts/api-versions/templates/_versions.tpl#L21-L27) (or vice versa).